sourcegraph/learn
# Remove description from frontend, keep as meta

### What should this PR do?
This resolves [DEVED-158](https://sourcegraph.atlassian.net/jira/software/projects/DEVED/boards/9?selectedIssue=DEVED-158). It removes the descriptions from the frontend of the site but keeps the meta descriptions in place.

### Why are we making this change?
This came from a conversation with Demand Gen. The descriptions are repetitive with the titles at the moment, it would be better if we could add more details / synonyms in there, but it would continue to clutter the page and may be confusing for users. Keeping descriptions as meta level and to support SEO / indexing is a common solution that we are going with here. 

### What are the acceptance criteria? 
Descriptions are no longer rendering on the frontend.
Site is working as normal. 

### How should this PR be tested?
Check out the branch and
* Ensure that descriptions are no longer rendering on the frontend.
* Ensure that the site is working as normal. 

## Pull request process

**Reviewers**:

1. Test functionality using the criteria above.
2. Offer tips for efficiency, feedback on best practices, and possible alternative approaches and things that may not have been considered.
3. For shorter, "quick" PRs, use your best judgement on #​2.
4. Use a collaborative approach and provide resources and/or context where appropriate.
5. Provide screenshots/grabs where appropriate to show findings during review.

**Reviewees**:

1. Prefer incremental and appropriately-scoped changes.
2. Leave a comment on things you want explicit feedback on.
3. Respond clearly to comments and questions.
